Tests/ScipioKitTests/DescriptionPackageTests.swift

```diff
@@ -65,17 +65,20 @@ final class DescriptionPackageTests: XCTestCase {
     }
 
     func testBuildProductsInCreateMode() throws {
-        let rootPath = fixturePath.appendingPathComponent("BinaryPackage")
+        let rootPath = fixturePath.appendingPathComponent("TestingPackage")
         let package = try XCTUnwrap(try DescriptionPackage(
             packageDirectory: rootPath.absolutePath,
             mode: .createPackage,
             onlyUseVersionsFromResolvedFile: false
         ))
-        XCTAssertEqual(package.name, "BinaryPackage")
+        XCTAssertEqual(package.name, "TestingPackage")
 
         XCTAssertEqual(
             Set(try package.resolveBuildProducts().map(\.target.name)),
-            ["SomeBinary"]
+            [
+                "Logging",
+                "TestingPackage"
```

Comment:
Could you just add a trailing comma here to follow the SwiftLint violation.

```suggestion
                "TestingPackage",
```

Reply:
Of course, it's fixed now.  I forgot to run the linter locally.